tests/config/cluster_install/ols_configmap.yaml

```diff
@@ -13,7 +13,9 @@ data:
   olsconfig.yaml: |
     llm_providers:
       - name: "$PROVIDER"
+        url: $PROVIDER_URL
```

Comment:
Need to mention  $PROVIDER_URL and $PROVIDER_DEPLOYMENT_NAME in the header ^^^

Reply:
fixed